### ISSUE TITLE
Discussion on final features and color palette for RollCall

### ISSUE THREAD

**Issue by Swarnimashukla** [Swarnimashukla/Automatic-attendance-management-system#107]
Hey all,

Now since we have wireframes  for different pages, logo designs, I would want to discuss on Final features and Color Palette we should be using while coding the website. 

Provide your suggestions in following manner:

1. Header ( sections that needs to be included e.g. Login/Signup button , About Us etc.)
2. Body (Main page features, you can divided it into sub-sections, as we do by creating containers while coding)
3. Footer (sections that needs to be included here e.g. Copyright, Social media , T&C) 

Do help us with finalizing color palette for website, considering this is an ed-tech project so take inspiration from Unacademy, Byjus and other ed-tech companies and provide your suggestions:

1. Primary color
2. Secondary color 
3. Font Style 

This issue is meant to finalize on features to begin with coding

**Comment by AasthaCodex** [comment]
hey cab you please assign this to me? i would like to give my suggestions.

**Comment by deeptea22** [comment]
@Swarnimashukla I would like to provide my suggestions on this. Where do i submit this?

**Comment by Swarnimashukla** [comment]
@deeptea22 @AasthaCodex 
 please share your suggestions here only in comments

**Comment by deeptea22** [comment]
Header - Logo, Website Name(Home Page), Search Bar, About Us, Contact Us, Login/Signup
Body   - Video/Photo Gallery, Features, Benefits, Achievements, Pricing, Testimonials
Footer - Copyright, Social Media Links, T&C

Primary Colour : #CFFE5
![image](https://user-images.githubusercontent.com/67323371/112022915-bc737c00-8b58-11eb-8764-37c408370ca9.png)

Secondary Colour: #FFE8DA
![image](https://user-images.githubusercontent.com/67323371/112023097-ed53b100-8b58-11eb-9a09-6c51d56184fe.png)

Font : Georgia

**Comment by Swarnimashukla** [comment]
![Group 1](https://user-images.githubusercontent.com/30214391/112101447-aa332580-8bcc-11eb-8348-a3facbbadb1b.png)


This could be used for primary and secondary color

**Comment by shwetaIAF** [comment]
> ![Group 1](https://user-images.githubusercontent.com/30214391/112101447-aa332580-8bcc-11eb-8348-a3facbbadb1b.png)
> 
> This could be used for primary and secondary 
![merge 1](https://user-images.githubusercontent.com/56163605/112760572-18d11280-9015-11eb-8ab0-056835c6dc6b.png)

**Comment by Abhishekkr93** [comment]
@Swarnimashukla Mam, which primary & secondary colour is final one? the finalized font is Georgia? I'm asking related to the footer issue. #118

**Comment by Swarnimashukla** [comment]
@Abhishekkr93  yes we can go with georgia for font and finalized colors are mentioned by me in comment section above

**Comment by Abhishekkr93** [comment]
@Swarnimashukla okay mam, finalized colors are Astros navy or azure?

**Comment by Swarnimashukla** [comment]
> @Swarnimashukla okay mam, finalized colors are Astros navy or azure?

For footer I think you can use  #2b3033(footer bg) and #f5f5f5 (for text) because above colors would be for rest of the page footer needs to be a little subtle only

**Comment by Abhishekkr93** [comment]
Okay mam, got it. I'll do the same.

**Comment by Sin-Sumit** [comment]
As the navbar issue is closed but I have some suggestions regarding the NAVBAR.
1. There should be some space left from the left and right side, and put all the options in the center where the logo is likely on the left and other things on the right with space between them.
2.  And it will be good to have only the Signup option rather than both, so on clicking that button, it will either take to a signup page where the login option is available there with the signup details or only use a pop-up method for signing or logging.
3.  And why is the search bar needed? If you help me in getting this please do.
4. Website name should be the heading of the homepage.
5. Thank you. Please let me know if it good or not.

**Comment by Sin-Sumit** [comment]
The UI theme is amazing from the navbar to the pricing section.

**Comment by Mehul-Jaiswal** [comment]
1) Home, Register, Log in, Core Team, Head teachers, Services, scholarships, careers(hiring)
2) 1st subsection: Welcome to <xyz> Company and a brief introduction about the company and in the side a register with us box. Use the contrast colour to highlight. I would suggest Teal(#158FAD) and Light Blue(#96C3EB)
After that have a section about the professors in different subjects and a short description about them. 
Next say what the students say about the teaching. Next number of students register and number of teachers you have.
3) In the footer have all social media hyperlinks.

COLOR:-
1)Primary Color: #1578ad
2) Secondary Color:#151dad
3) Font color: #2e353b
